Sources/SpeziAccessGuard/CodeViews/CodeView.swift

```diff
@@ -12,8 +12,10 @@ import SwiftUI
 
 struct CodeView: View {
     @Binding private var codeOption: CodeOptions
-    private let action: (String) async throws -> Void
+    private let action: (String, ValidationResult) async throws -> Void
+    private var toolbarButtonLabel: String = ""
```

Comment:
Can most likely be a constant.
We have been improving our localization in a few Spezi Packages over the last year or so and have generally moved to using [`LocalizedStringResource`](https://developer.apple.com/documentation/foundation/localizedstringresource)s to pass around localized strings. Maybe we should change this here and kickstart that as part of this improvement.
```suggestion
    private let toolbarButtonLabel: String = ""
```